services/node_util.go

```diff
@@ -114,7 +114,7 @@ func resetMaxUnavailable(maxUnavailable, lenInactiveHosts int, component string)
 	}
 
 	if lenInactiveHosts > 0 {
-		if maxUnavailable == lenInactiveHosts {
+		if lenInactiveHosts >= maxUnavailable {
 			return 0, fmt.Errorf("cannot proceed with upgrade of %s since %v host(s) are found to be inactive prior to upgrade", component, lenInactiveHosts)
```

Comment:
Nit: Is inactive the way we want to describe it? There are so many reasons why a host would not be available to upgrade. Or what if the upgrade failed?

Reply:
Could we update to indicate what "inactive" means in this case?